prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html

```diff
@@ -201,31 +205,34 @@
         <span>{{ AgreementType[agreement.agreementType] | case: 'snake' : 'space' | capitalize: true }}</span>
       </button>
     </mat-menu>
-
   </ng-template>
 
   <ng-template #siteMenu
                let-site="site"
                let-organizationId="organizationId">
-    <button mat-icon-button
-            [matMenuTriggerFor]="menu"
-            [disabled]="isPendingTransfer()">
-      <mat-icon>more_vert</mat-icon>
-    </button>
-
-    <mat-menu #menu="matMenu"
-              aria-label="Site actions menu">
-      <button mat-menu-item
+    <div class="mat-header-button-group">
+      <button mat-button
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Site Information"
+              [disabled]="isPendingTransfer()"
               (click)="viewSite(organizationId, site)">
-        <span>View/Update Site Information</span>
+        <mat-icon>folder</mat-icon>
```

Comment:
Just my subjective opinion, but `description` icon seems better

Reply:
Agree. It is updated.